app/controllers/request-to-go-live/index/get.controller.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+// Local dependencies
+const response = require('../../../utils/response')
+
+module.exports = (req, res) => {
+  const externalServiceId = req.service.externalId
+  return response.response(req, res, 'request-to-go-live/index', {externalServiceId})
```

Comment:
Just a quick note that it's generally considered bad practice to pass the `req` and `res` Express objects to other methods that aren't directly in the middleware stack. It makes it hard to reason about the request flow and can lead to accidentally terminating a response early or twice.

This is nothing to do with this PR or body of work I'm just noting to reference in future - I'll follow this up with a refactor ticket.

Reply:
Also while we're at it `response.response` is hard to read.

Reply:
Yes, as discussed with @sfount  we need to do this part of broader refactoring, for example not to pass `req` and `res` to "external" functionality and we should think of better naming as part of this refactoring.
This will be done after some research and reference implementation of `pay-toolbox`